sdk/keyvault/keyvault-keys/test/public/utils/common.ts

```diff
@@ -7,7 +7,7 @@ import { env } from "@azure-tools/test-recorder";
 /**
  * The known API versions that we support.
  */
-export const serviceVersions = ["7.0", "7.1", "7.2", "7.3", "7.4"] as const;
+export const serviceVersions = ["7.0", "7.1", "7.2", "7.3", "7.4", "7.5-preview.1"] as const;
```

Comment:
Not blocking, but do you not test the other packages against 7.5-preview.1? That's how we found problems with MHSM.

Reply:
The other packages already do, I just missed this one in my first PR.